packages/flutter_test/lib/src/widget_tester.dart

```diff
@@ -116,6 +117,11 @@ E? _lastWhereOrNull<E>(Iterable<E> list, bool Function(E) test) {
 /// If the [tags] are passed, they declare user-defined tags that are implemented by
 /// the `test` package.
 ///
+/// The argument [experimentalLeakTesting] is experimental and is not recommended
+/// for use outside of Flutter Framework.
+/// [experimentalLeakTesting] is noop.
+/// Use the method `testWidgetsWithLeakTracking` for leak free tests.
```

Comment:
This is a pkg:flutter internal method that we don't expose to the public, no? We probably shouldn't reference this in our public docs as "nobody" on the outside can really use this...

Reply:
1. The doc is saying: `is not recommended for use outside of Flutter Framework.`
2. This dos is for FF developers, who are supposed to use this parameter. 
3. It is temporary state: testWidgetsWithLeakTracking will be cleaned up very soon and the doc will be updated.